community-membership.md

```diff
@@ -113,26 +113,26 @@ only be doing issues triage and have no approval rights.
 ### Requirements
 
 The following apply to the part of the codebase for which one would be an
-approver in the `CODEOWNER` files.
+approver in the `CODEOWNERS` files.
 
-- Reviewer of the codebase for at least 3 months
-- Reviewer for at least 10 substantial PRs to the codebase
-- Reviewed or merged at least 30 PRs to the codebase
+- Reviewer of the codebase for at least 1 month
+- Reviewer for or contributor of at least 10 substantial PRs to the codebase
+- Reviewed or contributed at least 20 PRs to the codebase
```

Comment:
Is it important to us to have both the "10 substantial" and "20 of any size" criteria here? I think I would rather just consolidate on the "substantial" PRs and leave this line out, just to remove the sense of false precision in the words.

(This is not a strongly-held opinion)

Reply:
No, it really isn't. 10 substantial LGTM, but the fuzziness of definition means that maintainer's discretion applies.

Reply:
Cool. Well, if we're in the neighborhood, consider this one vote to consolidate these since it's already poorly defined and thus discretional.

Reply:
To avoid confusion in the future here, I'd also suggest adding details on both lines.

Reply:
I still vote for removing this line entirely and sticking with the "10 substantial PRs" on its own. That seems (way) more important, and it would make these rules simpler to explain and enforce as well.

Reply:
done.